dipy/io/tests/test_io_gradients.py

```diff
@@ -58,6 +59,14 @@ def test_read_bvals_bvecs():
     # You entered the bvecs on both sides:
     npt.assert_raises(IOError, read_bvals_bvecs, fbvecs, fbvecs)
 
+    # All possible delimiters should work
+    bv_file4 = tempfile.NamedTemporaryFile(mode='wt', delete=False)
+    bv_file4.write("66 55 33")
+    bv_file4.close()  # Does not close
+    bvals = read_bvals_bvecs(bv_file4.name, None)
```

Comment:
Should be `bvals, _ = read_bvals_bvecs(bv_file4.name, None)`.

Please, create a small `bvect` file too.